ui/pages/send/send-content/send-content.component.js

```diff
@@ -57,7 +57,7 @@ export default class SendContent extends Component {
           {this.maybeRenderAddContact()}
           <SendAssetRow />
           <SendAmountRow />
-          <SendGasRow />
+          {process.env.SHOW_EIP_1559_UI ? null : <SendGasRow />}
```

Comment:
In our new flow the gas is not editable on the send screen, edits will only happen on confirmations.